src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -4,9 +4,9 @@
 import mate.academy.model.CinemaHall;
 
 public interface CinemaHallService {
-    CinemaHall add(CinemaHall cinemaHall);
+    CinemaHall add(CinemaHall entity);
```

Comment:
why??? I don't get it. I recommend revert this
same in other service

Reply:
don't pay attention)

Reply:
I just wanted to use GenericService, but than changed my mind